src/audits/performance/first-meaningful-paint.js

```diff
@@ -61,13 +67,13 @@ class FirstMeaningfulPaint extends Audit {
           // The fundamental Time To fMP metric
           const firstMeaningfulPaint = fmp.firstMeaningfulPaint - fmp.navigationStart;
 
-          // Roughly an exponential curve.
-          //   < 1000ms: penalty=0
-          //   3000ms: penalty=90
-          //   >= 5000ms: penalty=100
-          const power = (firstMeaningfulPaint - 1000) * 0.001 * 0.5;
-          const penalty = power > 0 ? Math.pow(10, power) : 0;
-          let score = 100 - penalty;
+          // Use the CDF of a log-normal distribution for scoring.
+          //   < 1100ms: score≈100
+          //   4000ms: score=50
+          //   >= 14000ms: score≈0
+          const distribution =
+              TracingProcessor.getLogNormalDistribution(SCORING_MEDIAN, SCORING_FALLOFF);
+          let score = 100 * distribution.computeComplementaryPercentile(firstMeaningfulPaint);
 
           // Clamp the score to 0 <= x <= 100.
           score = Math.min(100, score);
```

Comment:
gotcha. I'm in favor of dropping it so that bugs are easier to identify. maybe the solution is `if (score <  0 || score > 100) throw new error`?